bulk-update/bulk-update.js

```diff
@@ -86,6 +86,22 @@ export async function loadListData(source, fetchFunction = fetch, fetchWaitMs =
   }
 }
 
+/**
+ * Generates the staged-content URL by localizing the stage path based on the entry path.
+ *
+ * @param {string} siteUrl - The base URL of the site.
+ * @param {string} entry - The entry path.
+ * @param {string} stagePath - The path to the stage.
+ * @param {string[]} locales - An array of supported locales.
+ * @returns {string} The staged URL.
+ */
+export function localizedStageUrl(siteUrl, entry, stagePath, locales = []) {
```

Comment:
What if there isn't a `stagePath`? 
```suggestion
export function localizedStageUrl(siteUrl, entry, stagePath = '', locales = []) {
```
Or could do an early return

Reply:
Empty string works for now, if we come up with more use cases we can decide if we want an early return.